cmd/tailscale/depaware.txt

```diff
@@ -98,6 +98,7 @@ tailscale.com/cmd/tailscale dependencies: (generated by github.com/tailscale/dep
         tailscale.com/internal/noiseconn                             from tailscale.com/cmd/tailscale/cli
         tailscale.com/ipn                                            from tailscale.com/client/tailscale+
         tailscale.com/ipn/ipnstate                                   from tailscale.com/client/tailscale+
+        tailscale.com/kube                                           from tailscale.com/envknob
```

Comment:
Hopefully this is OK, that package contains only some types and is intended for sharing with things that don't want to pull in upstream kube libraries